src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -398,6 +398,7 @@ void PerformInteractionRequest::ProcessUIResponse(
       application_manager_.hmi_interfaces().GetInterfaceState(
           HmiInterfaces::HMI_INTERFACE_UI);
   bool result = false;
+  // cppcheck-suppress redundantAssignment
   result = Compare<hmi_apis::Common_Result::eType, EQ, ONE>(
```

Comment:
bool result = Compare<hmi_

Reply:
Fixed in d4c6de32f5592edc35229f55afdec6c02c55b0e8
Please check